parent/pom.xml

```diff
@@ -153,6 +153,8 @@
 		<!-- discovery.tpc.version usually is usually set to ${tpc.version} but it
 		can happen that they need to have different values, hence 2 properties -->
 		<discovery.tpc.version>${TARGET_PLATFORM_CENTRAL_MAX}</discovery.tpc.version>
+		<testProduct>com.jboss.devstudio.core.product</testProduct>
+		<devstudio.repository>https://devstudio.redhat.com/10.0/snapshots/updates/</devstudio.repository>
```

Comment:
Does this URL ever change? Do you ever use the /stable/ URL, for example? If it's always /snapshots/ then I'm not sure we need it hardcoded in parent pom. Maybe set this in your jbosstools-integration-tests root pom instead?